travis_configure.sh

```diff
@@ -148,13 +131,26 @@ run_tests()
     return 0
 }
 
+# Invoked by the travis hook script
+#
+# - if the is run in docker stop containers and remove containers, networks,
+#   volumes, and images created by `up`.
+after_script()
+{
+    if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
+        docker-compose down
```

Comment:
Even when this might be nice, I guess it's not really necessary (at least for travis), the VM will be killed anyway.

Reply:
True, I will try to test if it all the same without `docker-compose down`